06-rmarkdown.Rmd

```diff
@@ -145,13 +171,13 @@ ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_
 
 The notation here means the following:
 
-- The three back ticks `"```"` indicate the barriers of where some code will be in between. This is what we call an R chunk. 
+- The three back ticks `"```"` indicate the barriers of where some code will be in between. This is what we call an R chunk.
```

Comment:
```suggestion
- The three back ticks `"```"` mark the boundaries of where code should be placed. This is what we call a code chunk.
```